grpc.sh

```diff
@@ -18,12 +18,16 @@ rsync -a $SOURCEDIR/ ./
 
 git submodule update --init # aie aie aie
 
-make ${JOBS:+-j$JOBS} prefix=$INSTALLROOT
+# gRPC has a custom Makefile which readily breaks with some environment vars, better to run it in a clean environment
+env -i HOME="$HOME" LC_CTYPE="${LC_ALL:-${LC_CTYPE:-$LANG}}" PATH="$PATH" USER="$USER" LD_LIBRARY_PATH="$LD_LIBRARY_PATH" make ${JOBS:+-j$JOBS} prefix=$INSTALLROOT
+
+# ldconfig won't work if we're not running make install as root, and that's ok, we don't need it
+sed -i 's/ldconfig/true/' ./Makefile
 make prefix=$INSTALLROOT install
 
 # Add missing symlink. Must be relative, because the directory is moved around after the build
 cd $INSTALLROOT/lib
-ln -s ./libgrpc++.so.1.2.5 ./libgrpc++.so.1
+ln -s ./libgrpc++.so.1.9.1 ./libgrpc++.so.1
```

Comment:
How about changing this to be version independent?

Reply:
It is not clear to me why this is even needed, I only updated it because it was there. The link with the correct soname (`libgrpc++.so.5` as well as `libgrpc++.so`) is already generated.

Reply:
Can you check and clean it up?

Reply:
Yep, I'm on it. I'll update this PR either with a version-independent `ln` or with a removal of that line.